builder/vsphere/driver/vm_cdrom.go

```diff
@@ -72,6 +73,30 @@ func (vm *VirtualMachineDriver) RemoveCdroms() error {
 	return nil
 }
 
+// RemoveNCdroms removes up to n CDRoms from the associated vm.
+// An error will occur If n is larger then the attached CDRoms.
+// n == 0 results in no CDRoms being removed
```

Comment:
```suggestion
// RemoveNCdroms removes up to n CD-ROMs from the image.
// An error will occur If n is larger then the attached CD-ROM count.
// n == 0 results in no CD-ROMs being removed.
```

Reply:
Done!

Reply:
Resolved in 860f7101405d68474e1acadf652ef640f3e43b2b.